scraper/src/github.py

```diff
@@ -159,6 +171,38 @@ def parse_event(self, event, event_time):
                 },
             )
 
+        elif event["type"] == "PushEvent":
+            branch = event["payload"]["ref"].split('/')[-1] 
+            commits = event["payload"]["commits"]
+            all_commits = []
+
+            # Iterate through each commit
+            for commit in commits:
+                # Fetch full commit details
+                commit_details = requests.get(commit["url"], headers=self.headers).json()  
+                
+                if len(commit_details.get("parents", [])) in [0, 1]:  
+                    # Prepare commit data
+                    commit_data = {
+                        "link": commit_details["html_url"],
+                        "text": commit["message"],
+                    }
+                    # Append commit data to the array
+                    all_commits.append(commit_data)  
+
+            # Append pushed commits event
+            self.append(
+                user,
+                {
+                    "type": "pushed_commits",
+                    "time": event_time,
+                    "title": f'Pushed({len(all_commits)}) commits to {event["repo"]["name"]}({branch})',
```

Comment:
Instead of storing repo and branch name in title, drop title field and instead store repo and branch as additional attributes.

This way if we ever want to rephrase the title for pushed_commits event, it'd just be a change in the nextjs app and we won't need to run a migration script.